src/components/mapView/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { MapContainer, Marker, TileLayer } from 'react-leaflet';
+import { Icon } from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+
+import useGeoLocation from 'hooks/useGeolocation';
+import locationIcon from 'assets/locationIcon.svg';
+
+import './styles.scss';
+
+const userLocationIcon = new Icon({
+  iconUrl: locationIcon,
+  iconRetinaUrl: locationIcon,
+  popupAnchor: [-0, -0],
+  iconSize: [53, 69],
+});
+
+const center = [-38.0174106, -57.6705735];
+
+const MapView = () => {
+  const location = useGeoLocation();
+  const { lat, lon } = location?.coordinates;
```

Comment:
This crashes the app if the user doesn't allow location access because the `onError` callback at `src/hooks/useGeolocation.js` updates a `location` state without a `coordinates` object inside.